### ISSUE TITLE
Browser

### ISSUE THREAD

**Issue by dragosstancu** [gwenaelp/vfg-field-array#8]
Hi,

First, thanks a lot for your Vue Form Generator plugin!

The thing is that I need this code to work on one page I'm using in a Node project.
I ran the built script, copied the files in my public folder and now I get an error when including the vfg-field-array.min.js. It looks like VueFormGenerator is not defined:
"Uncaught TypeError: Cannot read property 'abstractField' of undefined"

What am I missing? Thanks!

**Comment by dragosstancu** [comment]
<script src="https://cdnjs.cloudflare.com/ajax/libs/vue/2.5.3/vue.js"></script>   
    <script type="text/javascript" src="vfg-core.js"></script>
    <script type="text/javascript" src="vfg-field-array.js"></script>

**Comment by gwenaelp** [comment]
You might need to import files that are in the dist folder.
You are not using webpack in your project? I would advise you to do so, it would make libraries integration easier in general.

**Comment by dragosstancu** [comment]
Thanks. I'll take a look at what's in the dist folder. I just wanted to use VUE for one page. I don't want to refactor the whole thing. I could have done it with jQuery but I love how VUE gets it more reactive.

**Comment by gwenaelp** [comment]
Yes I know what you mean, but it might still be more painful to use vue without all the tooling usually setup to make your life easier ! But of course you can develop how you want, as long as you find it useful!

I hope you will find a way to make it work!

**Comment by dragosstancu** [comment]
I gave up, I think I needed to set up a project the right way and run build. 
It's funny because I got it to work with vue-form-generator but adding a custom component was a bummer. I'm coming from the jQuery ecosystem and it's a bit confusing and it feels like an overkill in terms of dependencies. I mean, I ran an "npm install" and suddenly I have thousands of files in modules.  I don't need the field array now but it looks like a big improvement. I've been using jQuery repeaters so far.
